apps/docs/pages/guides/auth/sessions.mdx

```diff
@@ -0,0 +1,103 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  id: 'sessions',
+  title: 'User Sessions',
+  description: 'Control and manage user sessions.',
+}
+
+Supabase Auth provides fine-grained control over your user's sessions.
+
+A session is created when a user signs in. It is represented by the Supabase Auth access token in the form of a JWT, and a refresh token which is a unique string.
+
+Access tokens are designed to be short lived, usually between 5 minutes and 1 hour. You can exchange a refresh token only once to get a new access and refresh token pair.
+
+This process is called **refreshing the session.** 
+
+By default, there's no limit to the number of sessions a user can have on Supabase Auth. This behavior can be controlled in the following ways.
+
+- By restricting the number of sessions a user can have.
+- By controlling the lifetime of a session.
+
+A session can terminate, depending on configuration, when:
+
+- The user clicks sign out.
+- The user changes their password or performs a security sensitive action.
+- It times out due to inactivity.
+- It reaches its maximum lifetime.
+- A new session for the same user was started.
+
+## Controlling the lifetime of a session
+
+<Admonition type="note">
+
+This feature is only available on Pro plans and up.
+
+</Admonition>
+
+Supabase Auth can be configured to limit the lifetime of a user's session. By default, all sessions are active until the user clicks log out or performs some other action that terminates a session.
+
+In some applications, it's useful or required from a security point of view to ensure that users authenticate often, or that sessions are not left active on devices for too long.
+
+There are two ways in which you can limit the lifetime of a session:
+
+- By enabling timeboxed sessions which terminate after a fixed amount of time.
+- By enabling inactivity timeout, such that if a session remains unrefreshed for too long it will be terminated.
```

Comment:
For the second point, do you mean that the session will only be revoked if there is no user activity detected after a certain time or do you mean it will be revoked if it hasn't been refreshed within the timeout? 

```suggestion
- By enabling time-boxed sessions which are revoked after a fixed amount of time.
- By enabling timeout after inactivity, such that a session will be revoked if no user activity has been detected.
```